index.html

```diff
@@ -10,10 +14,70 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+    <nav>
+        <img class="logo" src="img/karma-logo.svg" alt="logo" width="35"/>
+        
+        <ul>
+            <li><a href="#" class="active">Meet Karma</a></li>
+            <li><a href="#">How It Works</a></li>
+            <li><a href="#">Store</a></li>
+            <li><a href="#">Blog</a></li>
+            <li><a href="#">Login</a></li>
+        </ul>
+        <img id="hamburger" src="img/menu-hamburger.svg" alt=""/>
+    </nav>
+    <header class="hero">
```

Comment:
The header is the part at the very top -- the logo and navigation. By that definition, `hero` should never find itself in the header, really. It should be a normal section. But it is indeed the hero :)